app/controllers/client_api/application_controller.rb

```diff
@@ -0,0 +1,15 @@
+module ClientApi
+  class ApplicationController < ::ActionController::API
+    before_filter :authenticate!
+
+    protected
+
+    def authenticate!
+      render json: { error: 'Unauthorized' }, status: 401 unless api_token
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.